data/createPages/index.js

```diff
@@ -45,6 +46,14 @@ const createPages = async ({ graphql, actions: { createPage } }) => {
         edge.node.parentSlug,
         edge.node.version,
       );
+      // Check if script exists for page
```

Comment:
This should be split out into a separate method (it can be in this file) that returns a script